openlibrary/templates/work_search.html

```diff
@@ -6,7 +6,7 @@
     $if k != 'has_fulltext':
         $changequery(page=None,**{k:param.get(k, []) + [v]})
     $else:
-        $changequery(page=None,**{k:v})
+        $changequery(page=None,**{k:list(v)})
```

Comment:
This wasn't necessary anymore

```suggestion
        $changequery(page=None,**{k:v})
```